portals/devportal/source/src/app/components/Apis/Details/SolaceApi/SolaceTopicsInfo.jsx

```diff
@@ -0,0 +1,580 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+ *
+ * This software is the property of WSO2 Inc. and its suppliers, if any.
+ * Dissemination of any information or reproduction of any material contained
+ * herein is strictly forbidden, unless permitted by WSO2 in accordance with
+ * the WSO2 Commercial License available at http://wso2.com/licenses.
+ * For specific language governing the permissions and limitations under
+ * this license, please see the license as well as any agreement you’ve
+ * entered into with WSO2 governing the purchase of this software and any
+ * associated services.
+ */
+
+import Loading from 'AppComponents/Base/Loading/Loading';
```

Comment:
The order of imports should be first library imports, second the local file imports with webpack alias( the line we refer here), Finally the relative imports.

Reply:
Fixed with https://github.com/wso2/apim-apps/pull/81/commits/70c7422e78f3f2ac41db0abbab5b8ebcc8faa27c